ci/scripts/e2e-sink-test.sh

```
@@ -70,14 +70,26 @@ grep -Fx 'before update' "$HTTP_SINK_OUTPUT"
 grep -Fx 'put payload' "$HTTP_SINK_OUTPUT"
+grep -Fx 'patch payload' "$HTTP_SINK_OUTPUT"
 grep -q '"event"' "$HTTP_SINK_OUTPUT"
 grep -Fx 'dynamic url payload' "$HTTP_SINK_OUTPUT"
 grep -Fx 'dynamic url as select payload' "$HTTP_SINK_OUTPUT"
-# Exactly 1 line from ignore_delete test + 2 from varchar test (NULL was skipped) + 1 from PUT + 1 from jsonb + 2 from dynamic URL tests
-test "$(wc -l < "$HTTP_SINK_OUTPUT")" -eq 7
-# Verify the default method remains POST and an explicitly configured PUT is honored
+grep -Fx 'dynamic patch payload' "$HTTP_SINK_OUTPUT"
+grep -Fx 'dynamic custom method payload' "$HTTP_SINK_OUTPUT"
+grep -Fx 'dynamic url and method payload' "$HTTP_SINK_OUTPUT"
+! grep -F 'skipped invalid method payload' "$HTTP_SINK_OUTPUT"
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)